articles/mfa/guides/configure-phone.md

```diff
@@ -0,0 +1,128 @@
+
+5. Click **Save**.
+
+## Custom Phone Messaging providers
+
+Phone Messaging providers not currently integrated with Auth0 can be supported by using the [Send Phone Message](/hooks/extensibility-points/send-phone-message) Hook. To learn how to implement this in your MFA flow, check the examples for different providers below:
+
+* [Amazon SNS](/mfa/send-phone-message-hook-amazon-sns)
+* [Twilio](/mfa/send-phone-message-hook-twilio)
+* [Infobip](/mfa/send-phone-message-hook-infobip)
+* [TeleSign](/mfa/send-phone-message-hook-telesign)
+* [Vonage](/mfa/send-phone-message-hook-vonage)
+* [Esendex](/mfa/send-phone-message-hook-esendex)
+* [Mitto](/mfa/send-phone-message-hook-mitto)
+
+## Using the Management API to configure Voice or SMS
+
+You can use the Management API to configure which Message Delivery Methods are enabled by using the `/api/v2/guardian/factors/phone/message-types` endpoint. 
+
+The `messages_types` parameter is an array that can have ["sms"], ["voice"], or ["sms", "voice"]. You need a [Management API Token](https://auth0.com/docs/api/management/v2/tokens) with the `update:guardian_factors` scope as a Bearer Token to call the API:
+
+ ```har
+  {
+      "method": "PUT",
+      "url": "https://${account.namespace}/api/v2/guardian/factors/phone/message-types",
+      "headers": [
+        { "name": "Content-Type", "value": "application/json" },
+   	    { "name": "Authorization", "value": "Bearer MGMT_API_ACCESS_TOKEN" }
+      ],
+      "postData": {
+          "mimeType": "application/json",
+          "text": "{ \"message_types\": [\"sms\", \"voice\"] }"
+      }
+  }
+```
+
+## Security Considerations
+
+When using any phone messaging provider, you need to be aware that attackers abusing the signup flow could cause you financial damage.
+
+Auth0 will let a single user send up to 10 sms or voice messages per hour. To further protect your account, you can consider:
+
+- Enabling [Brute Force Protection](/anomaly-detection/references/brute-force-protection-triggers-actions#100-failed-login-attempts-or-50-sign-up-attempts). Auth0 will block an IP if it attempts to do more than 50 signup requests per minute.
+
+- Enable [Log Streaming](https://auth0.com/docs/logs/streams) and create alerts using your favorite monitoring tool, when you see spikes in the number of `gd_send_voice` or `gd_send_voice_failure` [log events](/logs/references/log-event-type-codes).
+
+Phone Messaging providers have additional protections. If you are using Twilio, make sure you read the [Anti-Fraud Developer Guide](https://www.twilio.com/docs/usage/anti-fraud-developer-guide). We recommend that you consider the following options:
+
+- Limit the countries that you'll send messages for [SMS](https://support.twilio.com/hc/en-us/articles/223181108-How-International-SMS-Permissions-work) and [Voice](https://support.twilio.com/hc/en-us/articles/223180228-International-Voice-Dialing-Geographic-Permissions-Geo-Permissions-and-How-They-Work).
```

Comment:
Let's add a bit of details why they should limit the number of enabled countries. Just add the sentence at the end of this paragraph like `, as there are countries with a higher risk of a [toll fraud](https://www.twilio.com/learn/voice-and-video/toll-fraud) and expensive calling rates.`